Sources/MealzUIModuleIOS/DefaultPageParameters/MyMeals/NestedMyMealsViewOptions.swift

```diff
@@ -18,15 +18,18 @@ import MealzIOSFramework
 @available(iOS 14, *)
 public struct NestedMyMealsViewOptions {
     public var title: TypeSafeBaseTitle
+    public var itemSelectorCTA: TypeSafeBaseButton
     public var recipeCard: TypeSafeMyMealRecipeCard
     public var recipeCardLoading: TypeSafeRecipeCardLoading
     
     public init(
         title: TypeSafeBaseTitle = TypeSafeBaseTitle(EmptyTitleView()),
+        itemSelectorCTA: TypeSafeBaseButton = TypeSafeBaseButton(MyBasketOpenItemSelectorCTA()),
```

Comment:
Is there a reason for it not to be named MealzMyBasketOpenItemSelectorCTA ?

Reply:
good catch